Dockerfile

```diff
@@ -1,23 +1,25 @@
-FROM openjdk:8
+FROM openjdk:17-jdk-slim-buster
 MAINTAINER Bruno De Bus <Bruno.DeBus@klarrio.com>
 
-ARG IMAGE_VERSION
-
+ARG image_version
```

Comment:
I would keep the uppercase ARG names.  I believe in docker files it is the more popular style.